src/index.js

```diff
@@ -97,9 +92,28 @@ function configuredFormatter(options = {}) {
     throw new Error(`${typeFormat} is not json or console.`);
 }
 
+function parseInfo(infoObj) {
+    return _.omit(infoObj, [
```

Comment:
Add code, if including in `err`

Reply:
We want `code` to _not_ get filtered out by `parseInfo()`. This is partially because
1. There is no way to determine if a member named `code` is a member of an error or of a different type of object passed into the logger.
2. If we filter out `code`, the logical extension of that would be to filter out any/every possible non-standard member of an object of type `Error`, which is impossible and also makes this library dependent on our error handling philosophy.
3. We do want `code` (and any other custom properties of objects of type `Error`) to show up in `meta.event`.